Makefile

```diff
@@ -48,63 +14,49 @@ TARGET_GOARCH=$(shell go env GOARCH)
-OPERATOR_IMAGE= $(REGISTRY)local-storage-operator:$(VERSION)
-MUST_GATHER_IMAGE = $(REGISTRY)local-must-gather:$(VERSION)
+IMAGE = $(REGISTRY)/local-volume-provisioner:$(VERSION)
+MUTABLE_IMAGE = $(REGISTRY)/local-volume-provisioner:$(VERSION)
+DISKMAKER_IMAGE = $(REGISTRY)/local-diskmaker:$(VERSION)
+OPERATOR_IMAGE= $(REGISTRY)/local-storage-operator:$(VERSION)
+MUST_GATHER_IMAGE = $(REGISTRY)/local-must-gather:$(VERSION)
 REV=$(shell git describe --long --tags --match='v*' --dirty 2>/dev/null || git rev-list -n1 HEAD)
 
-# Image URL to use all building/pushing image targets
-IMG ?= controller:latest
-
 # Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
 ifeq (,$(shell go env GOBIN))
 GOBIN=$(shell go env GOPATH)/bin
 else
 GOBIN=$(shell go env GOBIN)
 endif
 
-# all: build
-
-##@ General
-
-# The help target prints out all targets with their descriptions organized
-# beneath their categories. The categories are represented by '##@' and the
-# target descriptions by '##'. The awk commands is responsible for reading the
-# entire set of makefiles included in this invocation, looking for lines of the
-# file as xyz: ## something, and then pretty-format the target and help. Then,
-# if there's a line with ##@ something, that gets pretty-printed as a category.
-# More info on the usage of ANSI control characters for terminal formatting:
-# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
-# More info on the awk command:
-# http://linuxcommand.org/lc3_adv_awk.php
-
-# help: ## Display this help.
-# 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
-
-##@ Development
-
 update: rbac manifests generate fmt
+.PHONY: update
 
 verify:
 	./hack/verify-manifests.sh
+	./hack/verify-generate.sh
+	./hack/verify-gofmt.sh
+.PHONY: verify
 
 rbac: controller-gen ## Generate ClusterRole and Role objects.
 	$(CONTROLLER_GEN) rbac:roleName=local-storage-operator webhook paths="./controllers/..." output:artifacts:config=config/rbac
 	$(CONTROLLER_GEN) rbac:roleName=local-storage-admin paths="./diskmaker/controllers/..."  output:artifacts:config=config/rbac/diskmaker
+.PHONY: rbac
 
 manifests: controller-gen ## Generate CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=local-storage-operator crd paths="./api/..." output:artifacts:config=config/crd/bases
+.PHONY: manifests
 
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./api/..."
+.PHONY: generate
 
 fmt: ## Run go fmt against code.
 	go fmt ./...
+.PHONY: fmt
 
-# vet: ## Run go vet against code.
-# 	go vet ./...
+vet: ## Run go vet against code.
```

Comment:
This belongs to `16baf4a4` ("add go vet to verify target and fix go vet errors"), not to this commit. At this point nobody uses this target, and `go vet` reports errors anyway.

Reply:
I squashed these commits